features/checkout/paying_for_order/changing_payment_method_after_order_confirmation.feature

```diff
@@ -7,6 +7,8 @@ Feature: Changing the method after order confirmation
     Background:
         Given the store operates on a single channel in "United States"
         And there is a user "john@example.com" identified by "password123"
+        And the store has a payment method "PayPal Express Checkout" with a code "PAYPAL" and gateway factory "Paypal Express Checkout"
+        And gateway of this payment method is configured with username "TEST", password "TEST" and "TEST" signature
```

Comment:
Can we create some step that will do it by default? It does not have any business value